modules/cluster-logging-feature-reference.adoc

```diff
@@ -44,18 +44,14 @@ $ oc -n openshift-logging edit ClusterLogging instance
 
 [source,yaml]
 ----
-  apiVersion: "logging.openshift.io/v1"
-  kind: "ClusterLogging"
-  metadata:
-    name: "instance"
-    namespace: "openshift-logging"
-    annotations:
```

Comment:
when Vector is tech-preview in Logging 5.3,  the annotation must be specified to enabled  vector.   

Since 5.4 is EOL,  How about delete the tech-preview  information totally?

Reply:
Thanks